code/modules/recycling/conveyor.dm

```diff
@@ -284,8 +284,9 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 
 	else if(!user.combat_mode)
 		user.transferItemToLoc(attacking_item, drop_location())
-	else
-		return ..()
+	
+	return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
```

Comment:
?

Reply:
this is from a previous fix which got merged, i dont know how it got in. Since it was merged, i assume master will have the same change?